test/APITests.js

```diff
@@ -838,4 +838,66 @@ describe('API Tests', () => {
     //   // auth headers
     // });
   });
+
+  describe('Export SVG API tests', () => {
+    it('listens to create SVG events', done => {
+      [div, api] = createElementAndApi(simple1And2dAnnotations, {
+        editable: false,
+        bounded: true
+      });
+
+      api.on('createSVG', svg => {
+        expect(svg.children.length).toEqual(2);
+        done();
```

Comment:
done called before return?

Reply:
I agree that it looks strange, but if it is after the return it will never be called, correct?